mathics/builtin/comparison.py

```diff
@@ -349,17 +386,42 @@ def apply_other(self, x, y, evaluation):
 
 
 class Less(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'Less[$x$, $y$]'
+    <dt>'$x$ < $y$'
+        <dd>yields 'True' if $x$ is known to be less than $y$.
+    <dt>'$lhs$ < $rhs$'
+        <dd>represents the inequality $lhs$ < $rhs$.
+    </dl>
+    """
     operator = '<'
     sympy_name = 'StrictLessThan'
 
 
 class LessEqual(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'LessEqual[$x$, $y$]'
+    <dt>'$x$ <= $y$'
+        <dd>yields 'True' if $x$ is known to be less than or equal to $y$.
+    <dt>'$lhs$ <= $rhs$'
+        <dd>represents the inequality $lhs$ ≤ $rhs$.
```

Comment:
IMO the best way to fix this would be to fully support TeX-style math in `$...$`. Some functions could definitely benefit from this (e.g. `SingularValueDecomposition`), plus it helps if we want to say (for example) how the gamma function is defined.